package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "javascript-express-api-postgres",
   "version": "1.0.0",
-  "description": "",
+  "description": "Assignments are located [here](./Assignments.md).",
```

Comment:
Let's revert this file back to it's initial state.  We shouldn't be making changes here.

Reply:
Make package.json look exactly like this:

```
{
  "name": "javascript-express-api-postgres",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "axios": "^0.27.2",
    "cors": "^2.8.5",
    "currency-formatter": "^1.5.9",
    "express": "^4.18.1",
    "pg": "^8.7.3"
  }
}
```